projects/go-lib/src/lib/components/go-icon/go-icon.component.ts

```diff
@@ -1,10 +1,25 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 @Component({
   selector: 'go-icon',
-  templateUrl: 'go-icon.component.html',
-  styleUrls: ['go-icon.component.scss']
+  templateUrl: 'go-icon.component.html'
 })
 export class GoIconComponent {
   @Input() icon: string;
+  @Input() iconModifier: string;
+  @Input() iconClass: string;
+
+  public classObject(): object {
```

Comment:
We should probably think of a consistent, potentially more performant way to handle this sort of class manipulation.

Reply:
Yeah, I don't really love this implementation... but understand the reasoning for it and at the current point in time I can't come up with a better alternative.